lib/matplotlib/backends/__init__.py

```diff
@@ -10,8 +10,7 @@
 # ipython relies on interactive_bk being defined here
 from matplotlib.rcsetup import interactive_bk
 
-__all__ = ['backend','show','draw_if_interactive',
-           'new_figure_manager', 'backend_version']
+__all__ = list(map(str, ['backend', 'pylab_setup']))
```

Comment:
@tacaswell: the removed symbols are currently not in the module namespace. Those symbols are local to the function `pylab_setup` in that module. By not removing those symbols from `__all__`, we are preventing this from working: `from matplotlib.backends import *`. Do you think we should still export those symbols somehow?

@WeatherGod: The `list(map(str, ['foo', 'bar']))` dance is a compatibility hack for Python 2. Because we do `from __future__ import unicode_literals`, we end up breaking `__all__ = ['foo', 'bar']` in Python 2; namely, `__all__` cannot take `unicode` strings. Perhaps we can create a wrapper function somewhere?